rust/operator-binary/src/config.rs

```diff
@@ -0,0 +1,287 @@
+use crate::airflow_controller::{CERTS_DIR, SECRETS_DIR};
+use stackable_airflow_crd::{
+    AirflowClusterAuthenticationConfig, AirflowConfigOptions, LdapRolesSyncMoment,
+};
+use stackable_operator::commons::{
+    authentication::{AuthenticationClass, AuthenticationClassProvider},
+    ldap::LdapAuthenticationProvider,
+    tls::{CaCert, TlsVerification},
+};
+use std::collections::BTreeMap;
+
+pub const PYTHON_IMPORTS: &[&str] = &[
+    "import os",
+    "from airflow.www.fab_security.manager import (AUTH_DB, AUTH_LDAP, AUTH_OAUTH, AUTH_OID, AUTH_REMOTE_USER)",
+    "basedir = os.path.abspath(os.path.dirname(__file__))",
+    "WTF_CSRF_ENABLED = True",
```

Comment:
Any story behind this? Docs state `Set to False to disable all CSRF protection. Default is True.`.
Better safe than sorry or had any problems? :)

Reply:
It is the [default](https://github.com/apache/airflow/blob/main/airflow/config_templates/default_webserver_config.py) used by airflow, but I haven't been able to find an explanation as to why.

Reply:
ok thx!